src/pages/signIn/signUp.jsx

```diff
@@ -0,0 +1,90 @@
+import { Formik } from 'formik';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+import { getUserInfo } from '../../redux/actions/actionCreators';
+import { initValuesForSignUp, ValidSchemeForSignUp } from './validation';
+import { SignUpForm, SignForm, SignInButton, StyledLink, StyledTitle } from './styled';
+import ValidInput from './validInput';
+import { useMutationDataServer } from '../../hooks/mutation/dataServerMutation';
+
+const SignUp = () => {
+    const dispatch = useDispatch();
+
+    const mutation = useMutationDataServer();
+
+    const loadDataToServer = (dataToSend) => {
```

Comment:
which data are you loading here?! make correct names. I told you about it a few minutes yesterday, please, don't ignore such things